cmd_gen_finetuning.ipynb

```diff
@@ -51,23 +51,29 @@
    "source": [
     "## 2. Install Python requirements\n",
     "\n",
-    "The following `pip` commands will install Unsloth and other required Python packages when [Conda](https://anaconda.org/anaconda/conda) is used to manage your Python environment, which is the case for many Jupyter notebook runtimes.\n",
```

Comment:
Please revert the changes to the installation instructions.

The official Unsloth documentation must be the source of truth as the package is very sensitive to the environment it is installed in.

Additionally, the new installation instructions are too specific for your environment. The previous instructions were generic and more suitable for AWS SageMaker and GCP Vertex AI notebook instances.